openedx/core/djangoapps/xblock/rest_api/urls.py

```diff
@@ -19,6 +19,8 @@
     path('', views.block_metadata),
     # get/post full json fields of an XBlock:
     path('fields/', views.BlockFieldsView.as_view()),
+    # Get the OLX source code of the specified block
+    path('fields/olx/', views.get_block_olx_view),
```

Comment:
```suggestion
    path('olx/', views.get_block_olx_view),
```
OLX isn't really a "Field" so I think just `olx/` makes more sense.

Reply:
Updated https://github.com/openedx/edx-platform/pull/35932/commits/46ac9e2da497011bba8558679d890164c8948fb7